ffi/opaque/mono_pkg.yaml

```diff
@@ -0,0 +1,20 @@
+dart:
```

Comment:
@johnpryan I'm guessing `mono_pkg.yaml` at this level inside of a git repo doesn't make a lot of sense?